### DISCUSSION TITLE
getting json response from $fetch.raw returns an error

### DISCUSSION THREAD

**Discussion by jeepers3327** [nuxt/nuxt#24100] [Questions]
I have route handler which makes an external api call but stuck with an error `[nuxt] [request error] [unhandled] [500] The body has already been consumed..`

Heres my code:
```
export default defineEventHandler(async (event) => {
  const body = await readBody(event);
  const csrfResponse = await $fetch.raw<any>(
    "http://localhost:4000/api/csrf_token"
  );

  const data = await csrfResponse.json();
});
```

The compiler points the error to line const data = await csrfResponse.json(); but I'm not sure why it happen since this line is the only one using the csrfResponse.

I don't encounter the said error when using the built in `fetch`.
```
  const csrfResponse = await fetch("http://localhost:4000/api/csrf_token");

  const data = await csrfResponse.json();
```

**Comment by Kasopej** [comment]
Nuxt $fetch wrapper already reads the response (calls response.json()), in order to provide _data and error objects.. yeah it can be annoying
